config/system/default.yaml

```diff
@@ -46,7 +46,5 @@ usi_setup:
   url: localhost:5000
   rpc_timeout_sec: 20
 
-# default mininet subnet
-mininet_subnet:
-  base_ip: 10.20.0.0
-  prefix_length: 16
+# default test host subnet
```

Comment:
is this default or internal?  They have slightly different meanings -- just wondering about congruence.  I'm fine either way (don't actually know what it's used for), but would at least want the comment to match the field :-)

Reply:
This matches the default specified in code so I put the default in comment here. This is used when instantiating a mininet network. http://mininet.org/api/classmininet_1_1net_1_1Mininet.html#a1ed0f0c8ba06a398e02f3952cc4c8393
I thought of 1 issue where test hosts and devices' IPs may conflict. If mininet assigns test host an IP (it usually just increments from the subnet) and external DHCP server assigns the same IP to a device; this was why I made the mininet's subnet configurable.

Reply:
Re: default vs. internal -- it's very confusing as written so please update to make it more clear.

Don't the test hosts have to be on the same subnet as the device?  If not, then the device will try to send the test host traffic to the gateway mac address which would fail?  I guess I don't understand the scenario... Might need a conversation outside of this PR (not sure it directly impacts this PR either).